server/templates/static/datasets.html

```diff
@@ -422,210 +247,166 @@ <h3><a href="https://www.eia.gov/electricity/data/eia860/">Energy Information Ad
-          error,
-          and
-          review status of earthquakes of magnitude 3 onwards starting from 1900.
-        </p>
-        <p> Data made available under <a
-            href="https://www.usgs.gov/information-policies-and-instructions/copyrights-and-credits">USGS Copyrights
-            and
-            Credits
-            Terms of Service</a>.
-        </p>
-
-        <h3><a href="https://www.usgs.gov/core-science-systems/ngp/board-on-geographic-names/download-gnis-data">United
+		  
+        <h3><a href="https://www.usgs.gov">United States Geological
+		  Service</a></h3>
+		  <ul>
+        <li><a href="https://earthquake.usgs.gov/data/comcat/">USGS Advanced
+            National Seismic  System Comprehensive Earthquake Catalog (ComCat)</a>:
+        earthquake source parameters (e.g. hypocenters, magnitudes, phase picks and amplitudes)
+          and other   products (e.g. moment tensor solutions, macroseismic information, tectonic summaries, maps) produced by
+          contributing  seismic networks. Data Commons includes date, time, location, magnitudes, magnitude errors, depth, depth
+          error,          and          review status of earthquakes of magnitude 3 onwards starting from 1900.
+	    </li>
+
+        <li><a href="https://www.usgs.gov/core-science-systems/ngp/board-on-geographic-names/download-gnis-data">United
             States
-            Geological Survey (USGS) Geographic Names Information System (GNIS) - National Federal Codes</a></h3>
-        <p> The National Federal Codes dataset includes codes, names, coordinates, and more information for all "named
+            Geological Survey (USGS) Geographic Names Information System (GNIS) - National Federal Codes</a>
+         The National Federal Codes dataset includes codes, names, coordinates, and more information for all "named
           physical
           and cultural geographic features (except
-        </p>
-        <p> roads and highways) of the United States", maintained by <a
+        
+        roads and highways) of the United States", maintained by <a
             href="https://www.usgs.gov/core-science-systems/ngp/board-on-geographic-names">GNIS</a>. Data Commons uses
           this
           dataset to build containment relationships between places from the <a
             href="https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.html">US Census
-            Gazetteer</a> dataset.
-        </p>
-        <p> Data made available under <a
+	    Gazetteer</a> dataset.
+	    </li>
+	    </ul>
+        
+        <p> <a
             href="https://www.usgs.gov/information-policies-and-instructions/copyrights-and-credits">USGS Copyrights
-            and
-            Credits
-            Terms of Service</a>.
+            and Credits Terms of Service</a>.
         </p>
 
       </article> <!-- end of id="federal-govt-data" -->
       <article id="state-govt-data">
 
+	    <!--
```

Comment:
are we leaving this out on purpose as well?

Reply:
yes